Cargo.toml

```
@@ -35,7 +35,7 @@ circom-prover = "=0.1.1"
 default = ["serde"]
```

```suggestion
default = []
```
I would remove serde as a default feature. `serde_json` is the one needed for the proof generation, so I would remove it from this block and remove it's optional flag. 

Also, the `reqwest` dep is duplicated (it's only being used in the build file), and `tokio` and `ruint` are not being used.

---

Hey! Thank you very much for the review! 

I just removed `reqwest` from `[dependencies]` to only have it in `[build-dependencies]`.

I also removed `tokio` and `ruint` dependencies since they are not used. 

Regarding `serde`, you mean this:

```rs
# serde
serde = { version = "1", features = ["derive"]}
serde_json = "1"

# circom-prover
rust-witness = "0.1"
circom-prover = "=0.1.1"

[features]
default = []
serde = [
    "zk-kit-lean-imt/serde"  
]
```

---

Amazing! So I think `serde` can be optional, just not `serde_json` 

```rust
# serde
serde = { version = "1", features = ["derive"], optional = true }
serde_json = "1"

[features]
default = []
serde = [
    "dep:serde",
    "zk-kit-lean-imt/serde"  
]
```

---

Updated! Thank you very much!